plugins/extractit/help.md

```diff
@@ -377,7 +377,7 @@ This action extracts all IPv4 and IPv6 addresses from a string or file.
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
 |file|bytes|None|False|Input file as bytes, supports text and binary file types such as PDF, DOCX, XLSX, PPTX, ODT, ODP, ODS|None|MTk4LjUxLjEwMC4xMDAgYW5kIDIwMDE6ZGI4Ojg6NDo6MiBhcmUgc2FtcGxlIElQIGFkZHJlc3Nlcw==|
-|str|string|None|False|Input string|None|198.51.100.100 and 2001:db8:8:4::2 are sample IP addresses|
+|str|string|None|False|Input string|None|https://example.com and 2001:db8:8:4::2 are sample IP addresses|
```

Comment:
Same for IP address. It seems that plugin regenerate cmd changed all the elements here 😄